cylc/flow/scheduler.py

```diff
@@ -603,6 +603,9 @@ def log_start(self) -> None:
 
             # Note that the following lines must be present at the top of
             # the workflow log file for use in reference test runs.
+            LOG.info(
+                "Task log prefix: CYCLE/TASK[/JOB][(FLOWS)]:STATUS"
+            )
```

Comment:
What is this for/what does it mean?

Reply:
This is an attempt to document the log file format from within the log file with the aim of making it more intuitive to new users.

Not sure it worked.

Reply:
I though it would be helpful because the format changed significantly in this PR, and it varies a bit:

- pre-incremented submit numbers are no longer logged for waiting states
- flow numbers are only logged if new flows are generated
- dropped the in-situ documentation (`flows=1,2`)


```
 Task log prefix: CYCLE/TASK[/JOB][(FLOWS)]:STATUS
...
-  [1/foo:waiting] => waiting(queued)
...
-  [1/foo/01:preparing] => submitted
 ...
-  [1/foo/02(1,2):preparing] => submitted
```
I'll remove the line if the consensus is that it's useless or unhelpful ... but does it really matter that the first part is the standard task ID that is documented elsewhere?

Reply:
Unfortunately I don't think many will understand what this line means. It's probably better explained in the docs?

Reply:
OK, I'll remove it.  In that case, I think I'll restore the in-situ `flows=` string and take the verbosity hit, as that's the part that's less obvious.